### PR TITLE
handle stable/dev image release creation

### PR DESCRIPTION
**Pull request by Ankurk99**
Ref: #412

### CODE REVIEW

File: k3s/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# K3s Installation
+
+Instead of self-managed Kubernetes, you can set up K3s by running the following command.
+
+```text
+$ cd KubeArmor/contribution/k3s
```
Comment (nyrahul) [inline k3s/README.md]: Reference to kubearmor in auto-policy-discovery??

File: k3s/install_k3s.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2021 Authors of KubeArmor
```
Comment (nyrahul) [inline k3s/install_k3s.sh]:
```suggestion
# Copyright 2022 Authors of auto-policy-discovery
```

File: k3s/install_k3s.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+if [ -f $KUBECONFIG ]; then
+	KUBECONFIGBKP=$KUBEDIR/config.backup
+	echo "Found $KUBECONFIG already in place ... backing it up to $KUBECONFIGBKP"
+	cp $KUBECONFIG $KUBECONFIGBKP
+fi
+
+cp /etc/rancher/k3s/k3s.yaml $KUBEDIR/config 
+
+echo "wait for initialization"
+sleep 15
```
Comment (nyrahul) [inline k3s/install_k3s.sh]: Can we remove the sleep and add something [like this](https://github.com/kubearmor/KubeArmor/blob/eb71e99aebc4e1df2b01ad11f62fd0a82b5933f7/.github/workflows/latest-release.yml#L46) to check if the pods are ready?

**Review by nyrahul — CHANGES_REQUESTED** [review]

File: k3s/README.md (hunk 1 above)
Comment (Ankurk99) [reply k3s/README.md]: Oops, I copied the k8s setup from KubeArmor, will change it

File: k3s/install_k3s.sh (hunk 3 above)
Comment (Ankurk99) [reply k3s/install_k3s.sh]: It is waiting for k3s to start. I think we can do without it as the commands are run step-wise.

**Review by nyrahul — APPROVED** [review]

**Pull request merged**